torch/_dynamo/package.py

```diff
@@ -258,7 +258,7 @@ def _get_code_source(code: types.CodeType) -> tuple[str, str]:
             if not hasattr(toplevel, part):
                 _raise_resolution_error(code, toplevel)
             toplevel = getattr(toplevel, part)
-            if inspect.isfunction(toplevel):
+            if inspect.isfunction(toplevel) or inspect.ismethod(toplevel):
```

Comment:
Hm is `ismethod` the right thing to use here? Won't this return true for instance methods in addition to class methods?